Content.Client/Cargo/UI/BountyHistoryEntry.xaml

```diff
@@ -0,0 +1,26 @@
+﻿<BoxContainer xmlns="https://spacestation14.io"
+                xmlns:customControls="clr-namespace:Content.Client.Administration.UI.CustomControls"
+                Margin="10 10 10 0"
+                HorizontalExpand="True"
+                Visible="True">
+    <PanelContainer StyleClasses="AngleRect" HorizontalExpand="True">
+        <BoxContainer Orientation="Vertical"
+                      HorizontalExpand="True">
+            <BoxContainer Orientation="Horizontal">
+                <BoxContainer Orientation="Vertical" HorizontalExpand="True">
+                    <RichTextLabel Name="RewardLabel"/>
+                    <RichTextLabel Name="ManifestLabel"/>
+                </BoxContainer>
+                <Control MinWidth="10"/>
+                <BoxContainer Orientation="Vertical" MinWidth="120">
+                    <RichTextLabel Name="TimestampLabel" HorizontalAlignment="Right" Margin="0 0 5 0"/>
+                    <RichTextLabel Name="IdLabel" HorizontalAlignment="Right" Margin="0 0 5 0"/>
+                </BoxContainer>
+            </BoxContainer>
+            <customControls:HSeparator Margin="5 10 5 10"/>
+            <BoxContainer>
+                <RichTextLabel Name="NoticeLabel" />
+            </BoxContainer>
```

Comment:
I don't think this BoxContainer is necessary.

```suggestion
            <RichTextLabel Name="NoticeLabel" />
```

Reply:
It wasn't!